docs/contributing/testing.md

```diff
@@ -37,6 +37,8 @@ You should add an example to the review app if the existing examples do not refl
 
 If you've created a new component, create a new `src/govuk/<COMPONENT>/<COMPONENT>.yaml` file instead, where `<COMPONENT>` is the name of the component you've created.
 
+Use `hidden: true` in your example if you do not want to include the example in the review app. The example will still appear in our [test fixtures](http://frontend.design-system.service.gov.uk/testing-your-html/).
```

Comment:
I wonder if this new paragraph would make more sense being positioned after
> 2. Add or update examples in the examples list.

as it's also about adding examples. 

The 'If you've created a new component' paragraph feels like it starts a new topic as it's about adding a new file

Reply:
That makes senses, thanks - will tweak.